addons/website/views/website_templates.xml

```diff
@@ -131,20 +130,21 @@
                     <t t-raw="0"/>
                 </main>
                 <footer>
-                    <div id="footer_container">
+                    <div id="footer">
                     </div>
                 </footer>
             </div>
-
-            <t t-call-assets="web.assets_common" t-css="false"/>
-            <t t-call-assets="website.assets_frontend" t-css="false"/>
-            <script t-if="website and website.google_analytics_key">
+            <script t-if="website and website.google_analytics_key and not editable">
                 (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
                 (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
                 m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
                 })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
 
                 ga('create', _.str.trim('<t t-esc="website.google_analytics_key"/>'), 'auto');
+                ga_google_analytics_data = <t t-raw="json.dumps(website.google_analytics_data(main_object))"/>;
```

Comment:
Related to the above question, seems strange to have code that will do and returns nothing just to prepare the override of website_version.